src/components/dropdown/dropdown.tsx

```diff
@@ -197,16 +200,19 @@ export class Dropdown implements InteractiveComponent, OpenCloseComponent {
           ref={this.setMenuEl}
         >
           <div
+            aria-labelledby={`${guid}-menubutton`}
             class={{
               ["calcite-dropdown-content"]: true,
               [PopperCSS.animation]: true,
               [PopperCSS.animationActive]: active || open
             }}
+            id={`${guid}-menu`}
             onTransitionEnd={this.transitionEnd}
             ref={this.setScrollerEl}
+            role="menu"
```

Comment:
Similar to above, to accommodate JAWS (all browsers) and NVDA in Chrome:

```html
role="listbox"
```

Reply:
Listbox isn't the right role for this element. Listbox is a selectable form element whereas this is a menu.

Reply:
I think menu only applies when the dropdown group is selection mode: none, as there’s no concept of chosen / selected item, and the dropdown items are actions or links. 

For single / multiple select, it does seem like a different role than menu applies.

Reply:
I'd prefer not to change the role based on the selection type. That could get complicated.

If we change it to use listbox then it is really just another `select`/`combobox`

The menu role does have selectable options. Such as...

```
const itemRole = this.href
      ? null
      : this.selectionMode === "single"
      ? "menuitemradio"
      : this.selectionMode === "multi"
      ? "menuitemcheckbox"
      : "menuitem";
```